test/integration/cli-image-create-kvm.test.js

```diff
@@ -45,6 +44,13 @@ test('triton image ...', testOpts, function (suite) {
     var originInst;
     var img;
 
+    if (!h.CONFIG.kvmImage || !h.CONFIG.kvmPackage) {
+        suite.comment('CONFIG kvmImage and kvmPackage are required ' +
```

Comment:
When skipping a test - I prefer the 'SKIP:' prefix to be used (makes it easy/consistent to search for skipped tests):
```
  suite.comment('SKIP: CONFIG kvmImage and kvmPackage...
```

Reply:
Sounds good!

Reply:
Done